sshcode.go

```diff
@@ -140,13 +141,22 @@ func sshCode(host, dir string, o options) error {
 		flog.Info("synced extensions in %s", time.Since(start))
 	}
 
+	codeServerFlags := []string{
+		fmt.Sprintf("--bind-addr 127.0.0.1:%v", o.remotePort),
```

Comment:
It seems like the latest `code-server` is using `--bind-addr` instead of `--host` and `--port`.

Reply:
This is no longer relevant, because I've noticed that `sshcode` does not download exactly the latest `code-server`.